pubsub/integration_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
```suggestion
	"io/ioutil"
```
Goes with the other change suggested, but feel free to do them in your own single commit